mm2src/rpc/lp_commands/lp_commands.rs

```diff
@@ -6,6 +6,8 @@ use derive_more::Display;
 use http::StatusCode;
 use serde_json::Value as Json;
 
+// Start get_public_key rpc implementation
+pub type GetPublicKeyRpcResult<T> = Result<T, MmError<GetPublicKeyError>>;
```

Comment:
This is a good example why such comments may not actually fulfill their intended purpose.
`GetPublicKeyRpcResult` is also used in `get_public_key_hash`, so this comment should be moved to `get_public_key`:
```rust
// Start get_public_key rpc implementation
pub async fn get_public_key(ctx: MmArc, _req: Json) -> GetPublicKeyRpcResult<GetPublicKeyResponse> {
```

But then it just doesn't make sense to leave these comments :)
Could you please remove them?